internal/http/cloudevents.go

```diff
@@ -95,6 +70,16 @@ func (a *AsyncEventMgr) ParseEvent(event string) (*AsyncEvent, error) {
 	return &asyncEvent, nil
 }
 
-func (a *AsyncEventMgr) IsLogIdle() bool {
+func (a *AsyncEventParser) IsLogIdle() bool {
 	return time.Since(a.lastLogTime) > 30*time.Second
 }
+
+func parseLineSequence(cloudEvent *cloudevents.Event) (string, error) {
+	// The extension is not necessarily present, so ignore errors.
+	_lineseq, _ := cloudEvent.Context.GetExtension("linesequence")
```

Comment:
hmm.. I believe a variable starting with a underscore is discouraged? What about `ls` or just `l`?
```suggestion
	ls, _ := cloudEvent.Context.GetExtension("linesequence")
```

Reply:
I haven't see any official recommendations for or against prefixing vars with underscores in go (not to be confused with guidelines around underscores in file names). But something like `ls` works well here 👍